app/views/moderatorIndex.pug

```diff
@@ -0,0 +1,28 @@
+extends layout
+
+block scripts
+    script(src='/static/bundle.js')
+
+block navbar
+    nav
+        ul.list-unstyled
+            li
+                a(href='/moderator/create_user') Registrer bruker
+            li
+                a(href='/moderator/activate_user') Aktiver bruker
+            li
+                a(href='/moderator/change_card') Mistet kort
+            li
+                a(href='/moderator/deactivate_users') Deaktiver brukere
+
+block content
+    .center(ng-repeat='alert in alerts', ng-class='{"fade": alert.fade}')
+        uib-alert(ng-bind='alert.message', type='{{alert.type}}', close='alert.close()')
+    .container(ng-view='')
+
+block footer
+    a(href='#', ng-controller='logoutController', ng-click='logout()' tabindex='-1') Logg ut
+    |  | Made by
+    a(href='http://github.com/webkom' tabindex='-1')  webkom
```

Comment:
Should also mention Abakus, but out of scope of this PR.

Reply:
Could might as well add it in while we're here :)